app/spdb/build.sbt

```diff
@@ -502,6 +503,30 @@ def gnodb(version: Version) = AppConfig(
   )
 ) extending List(odbproduction(version), gnodb_credentials(version))
 
+// GN AGS ODB: hbfauxodb-lv1 / gnauxodb
+def gnagsodb(version: Version) = AppConfig(
```

Comment:
What kind of risk are you thinking?

This is actually necessary because the IP address needs to be set as it is in the configuration or the ODB will not allow connections.

I used the same configuration for most things as gnodbtest, which shouldn't do anything with external effects, and we won't have a crontab file launching any services, so I don't foresee any issues.